vagrant/base.sh

```diff
@@ -27,13 +27,9 @@ if [ -z `which javac` ]; then
     apt-get -y update
 
     # Try to share cache. See Vagrantfile for details
-    mkdir -p /var/cache/oracle-jdk7-installer
-    if [ -e "/tmp/oracle-jdk7-installer-cache/" ]; then
-        find /tmp/oracle-jdk7-installer-cache/ -not -empty -exec cp '{}' /var/cache/oracle-jdk7-installer/ \;
-    fi
-    if [ ! -e "/var/cache/oracle-jdk7-installer/jdk-7u80-linux-x64.tar.gz" ]; then
```

Comment:
Why don't we need this anymore?

Reply:
Somehow I thought it would be better to use the java8 installer instead of our cached version, but on second thought it makes sense since we start multiple machines

Reply:
I think I remember now. I added this back and tested with vagrant/vagrant-up.sh but what happens is the install oracle-java8-installer package downloads and installs jdk-8u171-linux-x64.tar.gz, so we'd have to update to this. However, if oracle-java8-installer changes which jdk it points to this will again become useless. My feeling is to just drop this pre-cache attempt entirely.

Reply:
After discussing offline the conclusion is it's safer to just switch to jdk8 for now and update jdk caching in a separate PR.